src/tasks/vesting_pool.ts

```diff
@@ -0,0 +1,215 @@
+        const targetAddress = ethers.utils.getCreate2Address(
+            deployerAddress,
+            encodedSalt,
+            ethers.utils.keccak256(deploymentCode)
+        )
+        console.log("Expected target address", targetAddress)
+        console.log("Transaction target", deployerAddress)
+        console.log("Transaction data", encodedSalt + deploymentCode.toString().slice(2))
+    });
+
+const calculateRequiredTokens = (inputs: { amount: string }[]): BigNumber => {
+    let sum = BigNumber.from(0)
+    for(const input of inputs) {
+        sum = sum.add(BigNumber.from(input.amount))
+    }
+    return sum
+}
+
+task("build_add_vestings_tx", "Creates a multisend transaction to assign multiple vestings based on a CSV")
+    .addFlag("transferTokens", "Indicate whether a the multisend should include the transfer of the required tokens")
+    .addFlag("createSafes", "Indicate whether a new Safe should be created for each vesting")
+    .addFlag("showVestings", "Indicate whether to display the information of the vestings that will be created")
+    .addPositionalParam("csv", "CSV file with the information of the Safes that should be created", undefined, types.inputFile)
+    .addParam("defaultOwner", "The default owner of the new vesting (or the Safe for the vesting) if no owner is specified", nameToAddress("Safe Foundation"), types.string, true)
+    .addParam("pool", "Address of the vesting pool", "", types.string)
+    .addParam("defaultManaged", "Flag if the vesting should be managed by default if not defined in CSV", false, types.boolean, true)
+    .addParam("defaultCurve", "Curve that should be used if not defined in CSV", "0", types.string, true)
+    .addParam("defaultStartDate", "Start date as unix timestamp that should be used if not defined in CSV", "", types.string, true)
+    .addParam("defaultDuration", "Duration in weeks that should be used if not defined in CSV", "", types.string, true)
+    .addParam("tokenAddress", "Token address that should be used for transfers (by default it tries to query this from the vesting pool)", "", types.string, true)
+    .addParam("tokenAmount", "Token amount that should be be transferred to vesting pool (by default it the required amount for the vestings to be created is used)", "", types.string, true)
+    .addParam("export", "If specified instead of executing the data will be exported as a json file for the transaction builder", undefined, types.string, true)
+    .setAction(async (taskArgs, hre) => {
+        if (!taskArgs.pool) throw Error("No vesting pool specified")
+
+        const inputs: { 
+            owner: string | undefined,
+            amount: string,
+            nonce: string,
+            curveType: string | undefined,
+            startDate: string | undefined,
+            duration: string | undefined,
+            managed: boolean | undefined,
+            expectedSafe: boolean | undefined
```

Comment:
```suggestion
            expectedSafe: string | undefined
```
I think this should be a string